### PR TITLE
added more info on Properties

### PR DESCRIPTION
**Pull request by Comprehensive-Jason**
- added explainer on using `"[[link]]"` quotes in YAML properties with internal links
- added explainer on _Date_ and _Date & Time_ properties with pictures showing how to change OS date format, hopefully providing a good resource to point people to
- fixed a few minor typos and grammar issues on the Properties page
- incorporated #552 (commit 79fe664)

### CODE REVIEW

File: en/Editing and formatting/Properties.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ You can change how properties are displayed in your note by going to  **Settings
 A few features are not supported in Obsidian:
 
 - **Nested properties** — to view nested properties we recommend using the Source display
-- **Bulk editing properties** — this can be achieved community-made tools such as Python scripts
+- **Bulk editing properties** — this can be achieved by community-made tools such as Python scripts
```
Comment (sigrunixia) [inline en/Editing and formatting/Properties.md:67]: Suggesting changing `by` to `with`

File: en/Attachments/Windows OS DateTime.png
Comment (claremacrae) [inline en/Attachments/Windows OS DateTime.png:1]: I welcome the screenshots. However, I think it's better for accessibility and readability for screenshots always to be in light mode.

File: en/Editing and formatting/Properties.md [hunk 2]
```diff
@@ -168,7 +194,7 @@ Obsidian comes with a set of default properties:
 
 ### Deprecated properties
 
-These properties are deprecated in 1.4. Please do not use them any more:
+These properties are deprecated in 1.4. Please do not use them anymore:
```
Comment (claremacrae) [inline en/Editing and formatting/Properties.md:197]: Hmm. I think the change may be technically correct - but to my English-speaking eye, the `anymore` looks like a typo - I don't remember ever seeing it written as one word,

https://www.scribbr.com/commonly-confused-words/anymore-vs-any-more/

File: en/Attachments/Windows OS DateTime.png
Comment (sigrunixia) [reply en/Attachments/Windows OS DateTime.png:1]: Not on this specific case, but is there a specification you are aware of that has guidelines on when to use light mode versus dark mode for accessibility purposes? Most information I have found focuses solely on the contrast.

File: en/Editing and formatting/Properties.md (hunk 2 above)
Comment (Comprehensive-Jason) [reply en/Editing and formatting/Properties.md:197]: Huh, I didn't know there was that big a difference between those two. In American English, _anymore_ is technically correct, while in UK English _any more_ is always used.

I guess the real question then is "Do the help docs conform to American English or UK English?"

Comment (Comprehensive-Jason) [reply en/Editing and formatting/Properties.md:197]: The help docs [Style Guide](https://help.obsidian.md/Contributing+to+Obsidian/Style+guide) say to follow the Google developer style guide, and the Google style guide says to prefer American spelling. 

Thus, I think _anymore_ is correct.

Comment (sigrunixia) [reply en/Editing and formatting/Properties.md:197]: Technically both can work, but `any more` sounds better to my American ears as while there is a `Please` in the sentence, it's more of an imperative. 

`He does not work here anymore`
versus
`Do we have any more cheesecake?`


Cambridge has a good comparison on the differences: https://dictionary.cambridge.org/grammar/british-grammar/any-more-or-anymore

File: en/Attachments/Windows OS DateTime.png
Comment (Comprehensive-Jason) [reply en/Attachments/Windows OS DateTime.png:1]: Screenshots changed to light mode.

**Pull request merged**